src/System.Security.Cryptography.Encoding/src/System/Security/Cryptography/Base64Transforms.cs

```diff
@@ -49,15 +50,15 @@ public byte[] TransformFinalBlock(byte[] inputBuffer, int inputOffset, int input
             return ConvertToBase64(inputBuffer, inputOffset, inputCount);
         }
 
-        private byte[] ConvertToBase64(byte[] inputBuffer, int inputOffset, int inputCount)
+        private static byte[] ConvertToBase64(byte[] inputBuffer, int inputOffset, int inputCount)
         {
-            char[] temp = new char[4];
-            Convert.ToBase64CharArray(inputBuffer, inputOffset, inputCount, temp, 0);
-            byte[] tempBytes = Encoding.ASCII.GetBytes(temp);
-            if (tempBytes.Length != 4)
+            Span<char> temp = stackalloc char[4];
+            Convert.TryToBase64Chars(inputBuffer.AsSpan(inputOffset, inputCount), temp, out int _);
+            Span<byte> bytes = stackalloc byte[4];
+            if (Encoding.ASCII.GetBytes(temp, bytes) != 4)
```

Comment:
`Encoding.ASCII.GetBytes(temp, bytes) != 4` will always evaluate to _false_. I'm not quite sure what this code was originally intended to do. Since this will require further investigation, for now I'd recommend reverting changes to this method.

Reply:
Reverted in https://github.com/dotnet/corefx/pull/36838/commits/1daf838d3f5a474856e9c3503604ba37758ad8fc